### PR TITLE
Add notice about X-Forwarded-For

### PR DESCRIPTION
**Pull request by LukasReschke**
@Xenopathic THX

### CODE REVIEW

**Comment by karlitschek** [comment]
makes sense :+1:

File: config/config.sample.php [hunk 1]
```diff
@@ -1087,6 +1087,9 @@
 
 /**
  * List of trusted proxy servers
+ * 
+ * If you configure these also consider setting `forwarded_for_headers` which
+ * otherwhise defaults to `HTTP_X_FORWARED_FOR`.
```
Comment (Xenopathic) [inline config/config.sample.php]: otherwise

Comment (Xenopathic) [inline config/config.sample.php]: Also, perhaps a quick addition ``... otherwise defaults to `HTTP_X_FORWARDED_FOR` (the `X-Forwarded-For` header)``

Comment (LukasReschke) [inline config/config.sample.php]: Done

File: config/config.sample.php [hunk 2]
```diff
@@ -1087,6 +1087,10 @@
 
 /**
  * List of trusted proxy servers
+ * 
+ * If you configure these also consider setting `forwarded_for_headers` which
+ * otherwise otherwise defaults to `HTTP_X_FORWARDED_FOR` (the `X-Forwarded-For`
```
Comment (MorrisJobke) [inline config/config.sample.php]: Let me fix the double `otherwise` here :)

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**